measCompApp/Db/measCompBinaryOut.template

```diff
@@ -1,6 +1,7 @@
 record(bo, "$(P)$(R)")
 {
-    field(PINI, "$(PINI=YES)")
+#   field(PINI, "$(PINI=YES)")
```

Comment:
You also break backwards compatibility by commenting out the line with $(PINI=YES).  If you don't want PINI then just set PINI=NO in you file that loads this.

Reply:
ok